core/src/main/java/io/grpc/internal/DelayedClientTransport.java

```diff
@@ -185,6 +187,13 @@ public final void ping(final PingCallback callback, Executor executor) {
     throw new UnsupportedOperationException("This method is not expected to be called");
   }
 
+  @Override
+  public Future<TransportTracer.Stats> getTransportStats() {
+    SettableFuture<TransportTracer.Stats> ret = SettableFuture.create();
+    ret.set(null);
+    return ret;
```

Comment:
Futures.immediateFuture(null)

Reply:
Futures is `@Beta` class so we can't use it.

Reply:
@carl-mastrangelo PTAL if you feel strongly about using immediateFuture, it can be copied into our internal utils class.